### ISSUE TITLE
Manually add new entries to reweight card

### ISSUE THREAD

**Issue by KlingFelix** [johannbrehmer/madminer#75]
Can we have an option that just manually add news new entries to the reweight card?

The application I have in mind is that then we can do signal and (same final state) backgrounds in the same run, with the same events, which is needed for MadFisher.

**Comment by johannbrehmer** [comment]
Good idea. We should probably address this simultaneously with additional, non-morphing benchmark points, see #60 .

**Comment by johannbrehmer** [comment]
#188 allows additional benchmarks (but not different processes / models / ...).
